ecms_base/modules/custom/ecms_workflow/ecms_workflow.post_update.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/**
+ * @file
+ * ecms_workflow.post_update.php
+ */
+
+declare(strict_types=1);
+
+use Drupal\ecms_workflow\EcmsWorkflowBundleCreate;
+
+/**
+ * Revoke permissions from non-drupal_admin roles.
+ */
+function ecms_workflow_post_update_revoke_emergency_notification_permissions(): void {
+  // @see RIGA-588.
+  // Revoke permissions from non-drupal_admin roles.
+  \Drupal::service('ecms_workflow.bundle_create')
+    ->revokeRolePermissions(
+      EcmsWorkflowBundleCreate::CONTENT_PUBLISHER_ROLE,
+      'emergency_notification'
+    );
+
+  \Drupal::service('ecms_workflow.bundle_create')
+    ->revokeRolePermissions(
+      EcmsWorkflowBundleCreate::CONTENT_AUTHOR_ROLE,
+      'emergency_notification'
+    );
+}
+
+/**
+ * Revoke permissions from non-drupal_admin roles.
+ */
+function ecms_workflow_post_update_unpublish_all_emergency_notifications(array &$sandbox): void {
```

Comment:
The comment doesn't seem to fit the function

Reply:
Fixed.